### PR TITLE
improved converters.ts doc

### PR DESCRIPTION
**Pull request by windushka**
aims to solve #166

### CODE REVIEW

File: src/utils/converters.ts [hunk 1]
```diff
@@ -3,9 +3,20 @@ import BigNumber from 'bignumber.js';
 const MASSA_SCALING_FACTOR = 9;
 
 /**
- * Convert any value expressed in Massa back to nanoMassa
- * @param amountInMassa any amount in Massa
- * @returns bigint representing the amount expressed in nanoMassa
+ * Convert any value expressed in Massa back to nanoMassa.
```
Comment (Elli610) [inline src/utils/converters.ts:6]: Is nanoMassa the smallest unit ? If so we can precise it

File: src/utils/converters.ts [hunk 2]
```diff
@@ -19,9 +30,20 @@ export const fromMAS = (
...
 
 /**
- * Convert any value expressed in nano Massa back to Massa
- * @param amountInNanoMassa any amount in nanoMassa
- * @returns BigNumber representing the amount expressed in Massa
+ * Convert any value expressed in nanoMassa back to Massa.
+ * This function takes an input value in nanoMassa and returns the equivalent value in Massa.
+ *
+ * @remarks
+ * Massa is a unit of currency and nanoMassa is a sub-unit, where 1 Massa equals 10^9 nanoMassa.
```
Comment (Elli610) [inline src/utils/converters.ts:37]: Is nanoMassa the smallest unit ? If so we can precise it

**Review by Elli610 — APPROVED** [review]

**Review by Elli610 — APPROVED** [review]

File: src/utils/converters.ts [hunk 3]
```diff
@@ -3,9 +3,20 @@ import BigNumber from 'bignumber.js';
...
- * @returns bigint representing the amount expressed in nanoMassa
+ * Convert any value expressed in Massa back to nanoMassa (the smallest unit of Massa).
+ * This function takes an input value in Massa and returns the equivalent value in nanoMassa.
+ *
+ * @remarks
+ * Massa is a unit of currency and nanoMassa is a sub-unit, where 1 Massa equals 10^9 nanoMassa.
+ * This function uses the BigNumber library to perform calculations with arbitrary precision.
+ *
+ * @param amountInMassa - The amount to convert, expressed in Massa. Can be a number, string, BigNumber, or bigint.
+ * @returns The amount expressed in nanoMassa, as a bigint.
```
Comment (0xSwapFeeder) [inline src/utils/converters.ts:14]:
```suggestion
 *
 * @returns The amount expressed in nanoMassa, as a bigint.
```

File: src/utils/converters.ts [hunk 4]
```diff
@@ -19,9 +30,20 @@ export const fromMAS = (
...
- * @returns BigNumber representing the amount expressed in Massa
+ * Convert any value expressed in nanoMassa back to Massa.
+ * This function takes an input value in nanoMassa and returns the equivalent value in Massa.
+ *
+ * @remarks
+ * Massa is a unit of currency and nanoMassa is a sub-unit, where 1 Massa equals 10^9 nanoMassa.
+ * This function uses the BigNumber library to perform calculations with arbitrary precision.
+ *
+ * @param amountInNanoMassa - The amount to convert, expressed in nanoMassa. Can be a number, string, BigNumber, or bigint.
+ * @returns The amount expressed in Massa, as a BigNumber.
```
Comment (0xSwapFeeder) [inline src/utils/converters.ts:41]:
```suggestion
 *
 * @returns The amount expressed in Massa, as a BigNumber.
```

**Review by 0xSwapFeeder — CHANGES_REQUESTED** [review]

**Review by 0xSwapFeeder — APPROVED** [review]
lgtm

**Review by Ben-Rey — APPROVED** [review]

**Pull request merged**